src/hooks.ts

```diff
@@ -0,0 +1,57 @@
+import { useEffect, useMemo, useState } from 'react';
+import { getTodos } from './api/todos';
+import { Todo } from './types';
+
+export const useGetTodos = (USER_ID: number, tempTodo: Todo | null) => {
```

Comment:
This hook is being used as a "logic container" so probably is better to use the `USER_ID` const inside this file and use all other functions that call the API here.
By doing this you won't need the control state anymore (isTodoDelete, handleTodoDelete...)

Reply:
USER_ID, isTodoDeleted and errorMessage is used in APP.tsx in many places. There are functions in semantic code (onClick itp.) that uses these consts, so it easier to change them via handleError and handleIsTodoDeleted